### PR TITLE
[Chronos] add multi-process dataloader support in chronos base_forecaster

### PR DESCRIPTION
**Pull request by ACupofAir**
## Description

This PR is for resolve the accuracy drop issue caused by multiprocessing training with batch size scaling.

* [x] add dataloader generator for multi-processes training in chronos base_forcasting

### CODE REVIEW

**Review by TheaperDeng — APPROVED** [review]
LGTM

**Review by plusbang — APPROVED** [review]
LGTM
